superset/config.py

```diff
@@ -188,7 +188,9 @@
 # ---------------------------------------------------
 # Feature flags that are on by default go here. Their
 # values can be overridden by those in super_config.py
-FEATURE_FLAGS = {}
+FEATURE_FLAGS = {
+    'ALL_COLS_FILTERABLE_GROUPABLE': False,
```

Comment:
@xtinec , Can you review this use of feature flags ?

Reply:
I'm not sure how these feature flags are supposed to work in practice in `superset_config`. It appears there's no good way to alter only one of them. You'd have to either alter all of them or none of them. If you were to copy/paste the whole dict to your `superset_config` you'd miss out on new ones as they come, and if you try to import the dict I think you'd run in circular import issue. @xtinec what do you think?

Reply:
@agrawaldevesh let's figure out what's up with postgres, let's not use the feature flag for this specifically or at least until we confirm it's usable. If you repush without the flag I can try and help debug the postgres-specific issue